packages/prediction-app/src/components/ChapValidator/ChapValidator.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from 'react'
+import i18n from '@dhis2/d2-i18n'
+import { Button, CircularLoader, IconError24 } from '@dhis2/ui'
+import { useNavigate } from 'react-router-dom'
+import styles from './ChapValidator.module.css'
+import { useChapStatus } from '../../features/settings/ChapSettings/hooks/useChapStatus'
+import { useRoute } from '../../hooks/useRoute'
+
+type Props = {
+    children: React.ReactNode
+}
+
+export const ChapValidator = ({ children }: Props) => {
+    const { route } = useRoute()
+    const { status, error, isLoading } = route ? useChapStatus({ route }) : { status: undefined, error: undefined, isLoading: false }
```

Comment:
Why did you add the type here? You should already be getting the type from the hook.